src/main/java/io/github/genomicdatainfrastructure/daam/model/RemsApplicationMapper.java

```diff
@@ -0,0 +1,144 @@
+// SPDX-FileCopyrightText: 2024 PNED G.I.E.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package io.github.genomicdatainfrastructure.daam.model;
+
+import io.github.genomicdatainfrastructure.daam.remote.rems.model.*;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+public class RemsApplicationMapper {
```

Comment:
Your suggestion definitely provides a better readability. 

I put Rems in the class name because my idea was to have this class dedicated to the mapping rems -> our own application entity. If we were to have another Rems-like provider, we would create another class to map their application entity to ours.

What do you think?